code/__HELPERS/unsorted.dm

```diff
@@ -1538,6 +1539,16 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 
 	SpinAnimation(rotation_speed, -1, clockwise, rotation_segments)
 
+	// Only create the list if we're adding a new item, so we don't have a bunch of empty lists on /every/ atom
+	if (!A.orbiters)
+		A.orbiters = list()
+
+	A.orbiters += (src)
+
+	// Since we're adding a ghost onto a list on any arbitrary atom, make sure we don't have any garbage collection
+	// issues if the ghost gets deleted
+	RegisterSignal(src, COMSIG_PARENT_QDELETING, .proc/stop_orbit)
```

Comment:
we don't care what happens to `src`. That's the atom doing the orbiting. We care about what it's orbiting, which is `A`.

Reply:
Fixed in b43e1b1